// Copyright (c) 2020-2021 The Open-Transactions developers
// This Source Code Form is subject to the terms of the Mozilla Public
// License, v. 2.0. If a copy of the MPL was not distributed with this
// file, You can obtain one at http://mozilla.org/MPL/2.0/.

syntax = "proto2";

package opentxs.proto;
option java_package = "org.opentransactions.proto";
option java_outer_classname = "OTLucreTokenData";
option optimize_for = LITE_RUNTIME;

import public "Ciphertext.proto";

message LucreTokenData {
    optional uint32 version = 1;
    optional Ciphertext privateprototoken = 2;
    optional Ciphertext publicprototoken = 3;
    optional bytes signature = 4;
    optional Ciphertext spendable = 5;
}
